proto/debugpb.proto

```diff
@@ -214,4 +214,6 @@ message GetRegionPropertiesRequest {
 
 message GetRegionPropertiesResponse {
     repeated Property props = 1;
-}
\ No newline at end of file
+    // The middle key of the region in encoded format, without ts padding.
+    bytes middle_key = 2;
```

Comment:
You can use 'middle_key_by_approximate_size' as an `props`'s name.

Reply:
Yes, so the PR can be closed now.